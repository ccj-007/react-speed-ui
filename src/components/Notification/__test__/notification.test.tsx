import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Notification } from '../../index';

describe('test Notification component', () => {
  it('should render', () => {
    const { asFragment } = render(
      <Notification
        data={{
          content: undefined,
          title: undefined,
          icon: undefined,
        }}
        width={0}
        height={0}
        position={'br'}
      />,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
