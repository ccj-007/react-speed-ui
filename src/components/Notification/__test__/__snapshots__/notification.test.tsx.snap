// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Notification component should render 1`] = `<DocumentFragment />`;
